stdlib/ballerina-builtin/src/main/ballerina/runtime/Package.md

```diff
@@ -0,0 +1,85 @@
+## Package overview
+
+This package includes functions to interact with the runtime, the invocation context and to manage errors.
+
+### Invocation Context
+
+The Invocation Context is a data holder which is created per request, and preserved for a single request-response flow. 
+The Invocation Context comprises of a unique id, UserPrincipal instance which includes user details and an AuthContext 
+instance which has the authentication related details if available.
+
+### Samples
+
+The following sample shows how to access the Invocation Context and set data to it.
+```ballerina
+import ballerina/runtime;
+
+// set date to the Invocation Context    
+// set the username ‘tom’ as the user id
+runtime:getInvocationContext().userPrincipal.username = "tom";
+// set claims
+map claims = {email:"tom@ballerina.com", org:"wso2"};
```

Comment:
Please format the code properly. It should be similar to '{ email: "tom@ballerina.com", org: "wso2" }'. Also, do make the map a constrained map with map<string>. Fix in all the places.